### PR TITLE
UI-3269: Update "add user" form and add new section "add device"

### PR DESCRIPTION
**Pull request by masmerino13**

### CODE REVIEW

File: submodules/users/views/creation.html [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
 		</div>
-
-		{{#unless _whitelabel.hide_user_passwords}}
-			<div class="control-group">
-				<label class="control-label" for="password"><i class="fa fa-unlock"></i></label>
-				<div class="controls">
-					<input required type="password" name="user.password" id="password" placeholder="{{i18n.users.dialogCreationUser.password}}"/>
+		<div class="row-fluid">
+			<div class="span6 to-left">
+				<label class="control-label" for="first_name">{{i18n.users.dialogCreationUser.lastName}}</label>
```
Comment (guillegr123) [inline submodules/users/views/creation.html]: Label `for` is related to the wrong input
```suggestion
				<label class="control-label" for="last_name">{{i18n.users.dialogCreationUser.lastName}}</label>
```

Also, you will need to set the `id` attribute for the input.

File: submodules/users/views/creation.html [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
-		<div class="control-group">
-			<label class="control-label" for="email"><i class="fa fa-key"></i></label>
-			<div class="controls">
+		<div class="row-fluid">
+			<div class="span6 to-left">
+				<label class="control-label" for="first_name">{{i18n.users.dialogCreationUser.firstName}}</label>
+				<input required type="text" id="first_name" name="user.first_name" placeholder="{{i18n.users.dialogCreationUser.firstName}}" />
+			</div>
+			<div class="span6 to-right">
+				<label class="control-label" for="email">{{i18n.users.dialogCreationUser.loginEmail}}</label>
```
Comment (guillegr123) [inline submodules/users/views/creation.html]: [The `for` attribute should match the input `id`](https://www.w3schools.com/tags/att_label_for.asp)
```suggestion
				<label class="control-label" for="username">{{i18n.users.dialogCreationUser.loginEmail}}</label>
```

File: submodules/users/users.js [hunk 3]
```diff
@@ -1872,6 +1958,25 @@ define(function(require) {
 			// If for some reason a vmbox number exist without an extension, we still don't want to let them set their extension number to that number.
 			allNumbers = arrayExtensions.concat(arrayVMBoxes);
 			formattedData.nextExtension = parseInt(monster.util.getNextExtension(allNumbers)) + '';
+			formattedData.listProvisioners = _.map(data.provisioners, function(brand) {
+				var models = _.chain(brand.families)
```
[the remark this answers is not in the document]
Comment (masmerino13) [reply submodules/users/users.js]: Nice, I like it!

Thank you!!

File: submodules/users/users.js [hunk 4]
```diff
@@ -1727,6 +1726,93 @@ define(function(require) {
...
+					selectedBrand = _.find(data.listProvisioners, function(brand) {
+						return brand.name === $brand;
+					});
+
+					$deviceModelSelect
+						.find('option')
+						.remove()
+						.end();
+
+					selectedBrand.models.map(function(model) {
```
[the remark this answers is not in the document]
Comment (masmerino13) [reply submodules/users/users.js]: Hmm i'm not sure about it

**Pull request merged**